cpp/conanfile.py

```diff
@@ -0,0 +1,15 @@
+from conans import ConanFile, CMake, tools
+
+
+class PlatformSettersConan(ConanFile):
+	settings = "os", "compiler", "build_type", "arch"
+	generators = "cmake"
+	
+	def requirements(self):
+		self.requires("gtest/cci.20210126")
+		self.requires("platform.interfaces/0.1.2")
+	
+	def build(self):
+		cmake = CMake(self)
+		cmake.configure()
+		cmake.build()
```

Comment:
```suggestion
[requires]
gtest/cci.20210126
platform.interfaces/0.1.3

[generators]
cmake
```
And rename to `conanfile.txt`